cfpb/consumerfinance.gov
# TDP Activity Search: Mobile customization

TDP activity search need to be customized since the desktop version of search had too many options. This will help folks move through the search more effectively when on mobile

---




## Additions
Added mobile classes to html
Added a rule for changing mobile facets

## How to test this PR

1. Go to the TDP Activity search, and scale down to mobile, then refresh the page, and notice if the facets work correctly.

## Notes and todos

Need to move the JS rule to somewhere more appropriate
-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
